src/pages/settings/Subscription/SubscriptionSettingsPage.tsx

```diff
@@ -1,15 +1,21 @@
-import React from 'react';
+import React, {useEffect} from 'react';
 import HeaderWithBackButton from '@components/HeaderWithBackButton';
 import * as Illustrations from '@components/Icon/Illustrations';
 import ScreenWrapper from '@components/ScreenWrapper';
 import useLocalize from '@hooks/useLocalize';
 import useWindowDimensions from '@hooks/useWindowDimensions';
 import Navigation from '@libs/Navigation/Navigation';
+import * as Subscription from '@userActions/Subscription';
+import SubscriptionPlan from './SubscriptionPlan';
 
 function SubscriptionSettingsPage() {
     const {isSmallScreenWidth} = useWindowDimensions();
     const {translate} = useLocalize();
 
+    useEffect(() => {
+        Subscription.openSubscriptionPage();
+    }, []);
+
     return (
         <ScreenWrapper testID={SubscriptionSettingsPage.displayName}>
```

Comment:
We need to account for not rendering info if a user is _not_ an owner of a workspace, showing a Not Found view if they navigate to it by direct URL. (Asked about it here: https://expensify.slack.com/archives/C036QM0SLJK/p1716996496210459?thread_ts=1716904586.883309&cid=C036QM0SLJK)

Reply:
How do you think we should approach that? I don't think it's possible to navigate directly to that screen. Should we do some conditional rendering in the navigator depending on if the user is a workspace owner?

Reply:
Done!

Reply:
Perfect